juwai/zask
# Make Python 3 compatible

## Summary of changes

Make the zask framework to be compatible with python27 / python3.6.

## How to Test

1. Build python2/3 virtual env.
```
    virtualenv .virtualenv -p /usr/local/bin/python3.6  (or 2.7)
```
2. Create file 'requirements.txt' for pip, and input the context as follow:
```
    gevent==1.3.7
    zerorpc==0.6.1
    SQLAlchemy==1.2.19
    pytest
```
3. Run command :  ve pip install -r requirements.txt
4. Run command:   ve python setup.py develop
5. Run test command:  ve py.test.   so should be no error there.


/fyi: @tjoelsson , @mark-juwai : Please review.

/cc: @xxh840912 @zhangjustin : Need your input regarding _concern_.
